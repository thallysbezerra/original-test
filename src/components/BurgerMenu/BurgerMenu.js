import React from 'react';
import PropTypes from 'prop-types';

import './BurgerMenu.scss';

const BurgerMenu = ({ onClick, status }) => (
	<button className="burger-menu" onClick={onClick}>
		<i className={`burger-menu__icon fas fa-${!status ? 'bars' : 'times'}`} />
	</button>
);

BurgerMenu.propTypes = {
	status: PropTypes.bool,
	onClick: PropTypes.func
};

export default BurgerMenu;
