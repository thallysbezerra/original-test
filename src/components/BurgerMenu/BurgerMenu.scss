@import '../../css/Theme.scss';

.burger-menu {
	background-color: transparent;
	border: none;
	font-size: 36px;
	height: 85px;
	outline: none;
	width: 85px;

	@include breakpoint(tabletL) {
		display: none;
	}
}
