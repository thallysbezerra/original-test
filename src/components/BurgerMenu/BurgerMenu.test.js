import React from 'react';
import BurgerMenu from './BurgerMenu';

import renderer from 'react-test-renderer';

describe('BurgerMenu UI', () => {
	it('<BurgerMenu /> must have the snapshot version.', () => {
		const tree = renderer.create(<BurgerMenu />);

		expect(tree.toJSON()).toMatchSnapshot();
	});
});
