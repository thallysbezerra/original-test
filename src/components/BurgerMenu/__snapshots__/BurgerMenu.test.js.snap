// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BurgerMenu UI <BurgerMenu /> must have the snapshot version. 1`] = `
<button
  className="burger-menu"
>
  <i
    className="burger-menu__icon fas fa-bars"
  />
</button>
`;
